### ISSUE TITLE
Accessibility change request, add 'box-shadow' property when using 'outline: 0;'

### ISSUE THREAD

**Issue by joseluisbango** [liferay/clay#4318]
-   [x] I have searched the [issues](https://github.com/liferay/clay/issues) of this repository and believe that this is not a duplicate.
- For Clay v2+, there is the issue: #4124

## Is there an example you can provide via codesandbox.com?
I can't provide an example via codesandbox.com

## What are the steps to reproduce?
In order to reproduce, you need to run this accessibility tool OAW described on https://issues.liferay.com/browse/LPS-133314.

The report is complaining about the outline rule: 'outline: 0;' because it thinks the keyboard focus indicator is not visible and this style makes it less accessible. 

https://github.com/liferay/clay/blob/1.x/src/scss/bootstrap/_dropdowns.scss#L27

## What is the expected result?
Have a clean accessibility report for our clients when using OAW tool which is owned by the Government of Spain and is used by any public institution to report and verify the validity of their site requested by law.

## Environment

| Tech  | Version |
| Liferay DXP | 7.0 |
| Clay  | v1  |

**Comment by pat270** [comment]
Closing this one, see https://github.com/liferay/clay/pull/4319#issuecomment-932236060
